crowbar_framework/app/models/ceph_service.rb

```diff
@@ -252,7 +252,7 @@ def validate_proposal_after_save proposal
     end
 
     unless radosgw_nodes.empty?
-      ProposalObject.find_proposals("swift").each {|p|
+      Proposal.where(barclamp: "swift").each {|p|
```

Comment:
You don't need all. Without all it's lazy loaded, but it doesn't matter anyway as the records are getting loaded when they are accessed like here with ```each```.